### ISSUE TITLE
a11y: Repeated IDs // repeated agency names in search filter

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#133]
## Description:
There are duplicated `id` attributes for elements on the search results page. This appears to be caused by agency names that are doubled-up in the agencies listing of checkboxes to filter results by. [Searching by "FedRAMP" will show this issue within results page.](https://all-sorns.app.cloud.gov/?search=fedramp)

## Screenshot:
![screenshot of duplicate filter agency names with IDs also duplicated](https://user-images.githubusercontent.com/1093423/105245764-8a7d7380-5b2f-11eb-8c18-5c2a3c9f2808.png)

## WCAG SC
[Success Criterion 4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html): In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.

## Recommended fix:
Ensure there are no duplicate `ID`s on the page. This will likely resolve itself if the agencies list does not contain duplicate agency names, as the `id` for each appear to be derived from the agency name.

**Comment by lauraGgit** [comment]
@iamjolly are you still experiencing this? I'm unable to reproduce

**Comment by iamjolly** [comment]
It looks resolved now!
